loginas/settings.py

```diff
@@ -17,6 +17,8 @@
 
 MESSAGE_LOGIN_REVERT = getattr(
     settings,
-    "MESSAGE_LOGIN_REVERT",
+    "LOGINAS_MESSAGE_LOGIN_REVERT",
     _("You are now logged back in as {username}")
 )
+
+MESSAGE_EXTRA_TAGS = getattr(settings, 'LOGINAS_MESSAGE_EXTRA_TAGS', '')
```

Comment:
@skorokithakis We only care about the settings file naming convention so that it won't conflict with other packages, we don't have to have a prefix `LOGINAS` for internal use.